R/prep_data.R

```diff
@@ -33,12 +33,15 @@
 #'   prepared. If training data is big, pull the recipe from the "recipe"
 #'   attribute of the prepped training data frame and pass that to this
 #'   argument. If present, all following arguments will be ignored.
-#' @param remove_near_zero_variance Logical. If TRUE (default), columns with
+#' @param remove_near_zero_variance Logical or list. If TRUE (default), columns
+#'   with
 #'   near-zero variance will be removed. These columns are either a single
 #'   value, or meet both of the following criteria: 1. they have very few unique
 #'   values relative to the number of samples and 2. the ratio of the frequency
 #'   of the most common value to the frequency of the second most common value
-#'   is large.
+#'   is large. If a list, it is passed to the `options` argument of
```

Comment:
I like the e.g. here, but instead of saying it's passed to `recipes::step_nzv`, let's say it's a list with these two items (can you provide only one?), explain what each of the values does including whether increasing each means getting rid of more or fewer columns, and provide default values.